lib/src/server/server.dart

```diff
@@ -135,10 +160,14 @@ class Server {
         _connections.remove(connection);
       });
     }, onError: (error) {
-      print('Socket error: $error');
+      _printSocketError(error);
```

Comment:
Maybe be this can just be `onError: _printSocketError`?

Reply:
That wasn't working before with the incorrect typing. But now that `_printSocketError` accepts an `Object`, it works. Updated.